app/src/main/java/com/datarockets/mnchkn/data/local/DatabaseHelper.kt

```diff
@@ -1,296 +1,105 @@
 package com.datarockets.mnchkn.data.local
 
-import android.content.ContentValues
+import com.datarockets.mnchkn.MunchkinApplication
+import com.datarockets.mnchkn.data.mapper.GameEntityMapper
+import com.datarockets.mnchkn.data.mapper.GameStepMapper
+import com.datarockets.mnchkn.data.mapper.PlayerEntityMapper
+import com.datarockets.mnchkn.data.mapper.PlayerMapper
 import com.datarockets.mnchkn.data.models.GameStep
 import com.datarockets.mnchkn.data.models.Player
-import com.squareup.sqlbrite.BriteDatabase
-import com.squareup.sqlbrite.SqlBrite
-import rx.Observable
-import rx.schedulers.Schedulers
-import timber.log.Timber
+import io.reactivex.Completable
+import io.reactivex.Maybe
+import io.reactivex.Single
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
 class DatabaseHelper
-@Inject constructor(dbOpenHelper: DbOpenHelper) {
+@Inject constructor() {
 
-    val briteDb: BriteDatabase
-
-    init {
-        val briteBuilder = SqlBrite.Builder()
-                .logger { message -> Timber.tag("Database").v(message) }
-        briteDb = briteBuilder.build().wrapDatabaseHelper(dbOpenHelper, Schedulers.immediate())
-        briteDb.setLoggingEnabled(true)
+    fun setPlayer(player: Player): Single<Player> = Single.create {
+        val playerId = MunchkinApplication.database.playerDao().setPlayer(PlayerEntityMapper.transform(player))
```

Comment:
@vshkl What about having private finalized `playerDao` and `gameDao` in database helper which you can pass as dependencies with Dagger? Just create methods `providesGameDao` and `providePlayerDao` in ApplicationModule. And pass `gameDao` and `playerDao` via constructor of database helper. Personally I don't like such a long call like `MunchkinApplication.database.playerDao` and bla-bla-bla.